src/D2L.CodeStyle.Analyzers/Immutability/UnsafeStaticsAnalyzer.cs

```diff
@@ -251,73 +242,18 @@ string fieldOrPropertyName
 		/// diagnostics.
 		/// </summary>
 		private IEnumerable<Diagnostic> GatherDiagnostics(
-			SemanticModel model,
 			MutabilityInspector inspector,
+			ISymbol fieldOrProperty,
+			ITypeSymbol fieldOrPropertyType,
 			Location location,
 			bool isStatic,
-			bool isReadOnly,
-			ITypeSymbol fieldOrPropertyType,
-			string fieldOrPropertyName,
-			ExpressionSyntax initializationExpression,
-			bool isProperty,
-			bool isAutoImplementedProperty
+			string fieldOrPropertyName
 		) {
 			if ( !isStatic ) {
 				yield break;
 			}
 
-			if( isProperty && !isAutoImplementedProperty ) {

```

Comment:
All of this logic is unnecessary because it's already implemented inside the mutability inspector (its here for historical reasons.)